lib/ar_lazy_preload/active_record/association.rb

```diff
@@ -7,5 +7,20 @@ def load_target
       owner.try_preload_lazily(reflection.name)
       super
     end
+
+    # rubocop:disable Metrics/AbcSize
+    def ids_reader
```

Comment:
I think we should patch `CollectionAssociation`, because `ids_reader` appears on that level (not in the association)

Reply:
Fixed.